assets/settings/default.json

```
@@ -680,7 +680,11 @@
       // 5. Never show the scrollbar:
       //    "never"
       "show": null
-    }
+    },
+    // Default depth to expand outline items when opening a file.
+    // Set to 0 to collapse all items that have children, 1 or higher to collapse items at that depth or deeper.
+    // Very large values (e.g., 100+) effectively expand all items.
+    "default_outline_expansion_depth": 100
```

```suggestion
    "expand_outlines_with_depth": 100
```

Not sure what is `default` in this story.